README.md

```diff
@@ -125,10 +125,14 @@ Transitions obey the following rules:
 * a `onMatch(change, db, auditDb, callback)` function than will run on changes
   that pass the filter.
 
-* accepts a document as a reference and makes changes using that reference,
-  copying is discouraged.
+* has an `onChange(change, db, audit, callback)` function that makes changes to
+  the `change.doc` reference (copying is discouraged). `db` and `audit` are
+  handles to let you query those DBs. More about `callback` below.
+
+* takes responsibility for saving the document and re-attaching the newly saved
+  document (with new seq etc) at `change.doc`
   
-* does not have side effects outside of the document passed in.  This might be
+* does not have side effects outside of `change.doc`.  This might be
```

Comment:
This isn't true any more is it? We do things like create patients. Maybe say that "ideally" it would have no side effects?